### ISSUE TITLE
Unable to disable GPU rendering

### ISSUE THREAD

**Issue by ankar84** [RocketChat/Rocket.Chat.Electron#2314]
**Describe the bug**
When user try to disable GPU rendering via File - Disable GPU or via command-line flag `--disable-gpu` app crashes

What operating system and which version?
Windows 10 
Which version of Rocket.Chat (Server)?
4.3.3
Which version of Rocket.Chat.Electron (Electron/Desktop)?
3.7.7
Is there any setting relevant changed?
No

**To Reproduce**
1. Start rocket.chat.exe with `--disable-gpu` flag
2. Get error

**Expected behavior**
Be able to disable GPU

**Screenshots**
No

**Additional context**
```
(node:17644) UnhandledPromiseRejectionWarning: Error: app.disableHardwareAcceleration() can only be called before app is ready
    at performElectronStartup (C:\Program Files\Rocket.Chat\resources\app.asar\app\main.js:1166:22)
    at C:\Program Files\Rocket.Chat\resources\app.asar\app\main.js:3872:5
    at Generator.next (<anonymous>)
    at fulfilled (C:\Program Files\Rocket.Chat\resources\app.asar\node_modules\tslib\tslib.js:114:62)
(Use `Rocket.Chat --trace-warnings ...` to show where the warning was created)
(node:17644) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)

```

**Comment by jeanfbrito** [comment]
We will take a look on this soon. It's on track already. Thanks for showing us!

**Comment by ankar84** [comment]
Fixed in 3.7.8!
